docs/security/security.md

```diff
@@ -1,3 +1,71 @@
 # Security
 
 ## Overview
+Security is strongest from conscious design at the outset, rather than retroactively trying to fill gaps after a solution has been built. In this respect, security should be considered at every stage of the applications lifecycle.
```

Comment:
we call this `SPbD` (security and privacy by design) - https://www.redbooks.ibm.com/abstracts/redp4641.html

Reply:
🙏 included the details and link